src/db/books.js

```diff
@@ -0,0 +1,59 @@
+// Code dealing with getting our books from a data source
+// 
+
+/**
+ * @typedef {import('./book').Book} Book
+ */
+
+module.exports = {
```

Comment:
So basically, this is just the same as the old db.js, but you pass the database instance to `initBooksDb` and get a real database instead of reading and writing a plain-old json file.